src/app/dim-ui/PageLoading.tsx

```diff
@@ -1,40 +1,42 @@
 import { RootState } from 'app/store/types';
 import clsx from 'clsx';
+import { Orchestration, Tween, Variants, motion } from 'framer-motion';
 import _ from 'lodash';
 import { useRef } from 'react';
 import { useSelector } from 'react-redux';
-import { CSSTransition, TransitionGroup } from 'react-transition-group';
 import { Loading } from './Loading';
 import styles from './PageLoading.m.scss';
 
 const messageSelector = (state: RootState) => _.last(state.shell.loadingMessages);
 
-const transitionClasses = {
-  enter: styles.pageLoadingEnter,
-  enterActive: styles.pageLoadingEnterActive,
-  exit: styles.pageLoadingExit,
-  exitActive: styles.pageLoadingExitActive,
+const animateVariants: Variants = {
+  initial: { opacity: 0 },
+  open: { opacity: 1 },
+};
+const animateTransition: Tween & Orchestration = {
+  duration: 0.1,
+  delay: 0.5,
+  ease: 'easeIn',
 };
 
 /**
- * This displays the page-level loading screen. React Suspense can make this obsolete once it's available.
+ * This displays the page-level loading screen.
  */
 export default function PageLoading() {
   const message = useSelector(messageSelector);
   const nodeRef = useRef<HTMLDivElement>(null);
   return (
-    <TransitionGroup component={null}>
-      {message !== undefined && (
-        <CSSTransition
-          nodeRef={nodeRef}
-          classNames={transitionClasses}
-          timeout={{ enter: 600, exit: 300 }}
-        >
-          <div ref={nodeRef} className={clsx('dim-page', styles.pageLoading)}>
-            <Loading message={message} />
-          </div>
-        </CSSTransition>
-      )}
-    </TransitionGroup>
+    message !== undefined && (
```

Comment:
I don't think an empty message would be a thing?

```suggestion
    Boolean(message) && (
```